homeassistant/components/blink/strings.json

```diff
@@ -128,6 +137,9 @@
     },
     "failed_disarm_motion": {
       "message": "Blink failed to disarm camera motion detection."
+    },
+    "service_removed": {
+      "message": "The service {service_name} has been removed. Please use the 2FA flow in the Blink integration configuration instead."
```

Comment:
I am not sure what `the 2FA flow` should mean for a user. I think we should explain that this has changed and that this service isn't needed anymore and that if there's a new OTP required, the user will be asked to reauthenticate